.github/renovate.json

```diff
@@ -1,13 +1,21 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", "schedule:monthly", ":maintainLockFilesWeekly"],
-  "separateMajorMinor": true,
+  "extends": [
+    "config:base",
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":maintainLockFilesWeekly",
+    ":prConcurrentLimit10",
```

Comment:
Change the limit to 3 since we still don't have mac runners, and this would consume the whole orgs runners.

Reply:
My take on us using this as the default is that it will be the norm for smaller repos, and I consider the SDK one of those. The large repos e.g. `clients` or `server` need more restriction. Remember that this will only run on the weekends too; we can't stop rebase events creating more runs at any time however.

Reply:
We can look into it after we have enterprise, until then we can't have 10 renovate PRs since that would consume all mac runners indefinitely.

Reply:
There's over a dozen configs with this already deployed ... any idea how many dependencies need updating here? As I've deployed this around the organization I usually only see it creating a handful and being done.

Reply:
No idea. We should strive towards staying on lower versions because we want to encourage adoption of the sdk. Which means we should have broad versions. I kind of don't really care about staying up to date assuming the lockfile refreshes us to the latest version.